webapp/src/Controller/Jury/ProblemController.php

```diff
@@ -900,7 +900,7 @@ public function fetchTestcaseAction(int $probId, int $rank, string $type)
             ->join('tc.content', 'tcc')
             ->select('tc', 'tcc')
             ->andWhere('tc.problem = :problem')
-            ->andWhere('tc.ranknumber = :ranknumber')
+            ->andWhere('tc.rank = :ranknumber')
```

Comment:
Shouldn't this be ranknumber?

Reply:
I assume this works because there's `getRank`. Let me try with `ranknumber`.